roles/kubernetes-apps/csi_driver/azuredisk/tasks/main.yml

```diff
@@ -25,8 +25,8 @@
     - {name: azure-csi-cloud-config-secret, file: azure-csi-cloud-config-secret.yml}
     - {name: azure-csi-azuredisk-controller, file: azure-csi-azuredisk-controller-rbac.yml}
     - {name: azure-csi-azuredisk-controller, file: azure-csi-azuredisk-controller.yml}
+    - {name: azure-csi-azuredisk-node-rbac, file: azure-csi-azuredisk-node-rbac.yml}
     - {name: azure-csi-azuredisk-node, file: azure-csi-azuredisk-node.yml}
-    - {name: azure-csi-node-info-crd.yml.j2, file: azure-csi-node-info-crd.yml}
```

Comment:
Why drop the CRD file from the list but not drop the CRD file itself from this commit?

Reply:
Because I haven't found this file in kubespray repository. It's just not there, hence the removal from the list (it was causing failure).